.git_hooks/pre-push

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+if ! docker-compose up -d db s3 ; then
```

Comment:
Hmm, my only concern is that this breaks workflows for people not using docker-compose ... but I'm not sure anyone is doing that? cc @rust-lang/docs-rs, are you happy with this?

Reply:
I know I use containers for the side services, but this may re-spawn them if they're already running

Reply:
IIRC when I ran `docker-compose up -d db s3` twice it just said it's already running.

We can put this conditional on some ENVAR you can set to enable it and keep `cargo run` as default.

Reply:
I think just leaving service management up to the user is the best thing to do

Reply:
Done.